docker-compose.yml

```diff
@@ -194,6 +196,8 @@ services:
       PGDATA: /chainstate/pg-data
 
   stacks-api:
+    profiles:
+      - disabled
```

Comment:
Will this approach need larger changes to make the API work e2e?

Reply:
This was a debug change that shouldn't have been committed, reverted